<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoomPopup</class>
 <widget class="QWidget" name="ZoomPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnZoomOut">
     <property name="toolTip">
      <string>zoom in</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/icons/resources/imgs/zoom_out_64px.png</normaloff>:/icons/resources/imgs/zoom_out_64px.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnZoomIn">
     <property name="toolTip">
      <string>zoom out</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/icons/resources/imgs/zoom_in_64px.png</normaloff>:/icons/resources/imgs/zoom_in_64px.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnReset">
     <property name="toolTip">
      <string>reset zoom level</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
padding:6px;
border-radius:4px;
background-color:#D3D3D3;
}

QPushButton:hover{
background-color:#C0C0C0;
}</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
